README.md

```diff
@@ -30,36 +58,84 @@ Or if you want developer version directly from github:
 rust-sc2 = { git = "https://github.com/UltraMachine/rust-sc2" }
 ```
 
-The simplest competetive bot in less than 30 lines:
+
+
+## StarCraft II
+
+### Windows and macOS
+
+Install SC2 through [Battle.net](https://www.blizzard.com/en-us/apps/battle.net/desktop).
+
+### Linux
+
+#### Lutris and Wine
+
+1. Install Lutris from your package manager
+2. [Install Battle.net dependencies](https://github.com/lutris/docs/blob/master/Battle.Net.md). (Wine and Vulkan drivers)
+3. [Install SC2 through Lutris](https://lutris.net/games/starcraft-ii/)
+
+#### Headless (no graphics)
+
+1. Download most recent [Linux Package](https://github.com/Blizzard/s2client-proto#linux-packages) (Maps will come with the zip)
+
+2. Unzip to ~/StarCraftII (you'll need the End User License Agreement Password above the Linux Packages link)
+
+# Bug Workarounds (as of 7/31/22)
```

Comment:
still have these issues?

Reply:
Hi Ultra,

1 and 2 are same behavior, doesn't expand `~` and doesn't recurse down Maps directory.

I believe [this commit](https://github.com/UltraMachine/rust-sc2/commit/72a716828aa290918f766f759ae498e8d2e857b7) causes a new error. When trying to run headful (setting sc2path and using wine_sc2 features), wine throws an error  `Critical error during loading process. Required library icuuc52.dll does not exist` but it exists in `Support64`. 
But I guess it could be one of your other commits changing directory behavio

Reply:
When I revert those changes to before this commit: https://github.com/UltraMachine/rust-sc2/commit/321a5e7c768d89f523d8447c4f3f9f161c6dd461

headful loads successfully. 

Headless loads successfully and doesn't crash with your new changes. (but still same `~` and `Maps` error)

Reply:
unfortunately this commit has nothing to do with new error, but I guess this will fix it: ce51454
also `~` expansion should work now.
regarding maps: try setting map name as `child_dir/map_name`. I don't plan adding maps recursion in meantime, since idk how to handle situation when there are 2 maps with same name but different dirs, should program use `a/map` or `b/map`, its not fact these 2 are same files, can be 2 different, can be more than 2 :confused:

Reply:
lemme know if issue fixed or not

Reply:
I am not sure what you mean by this: `regarding maps: try setting map name as child_dir/map_name.`  When you download the linux client, it comes with the dir structure as:
`StarCraftII/Maps/LadderXXXXSeasonX/*.SC2Map` . So you are saying, change the map name to `LadderXXXSeasonX/<map>.SC2Map`? I don't think that is what you mean. 

Either way, I get what you mean. I see that there are multiple versions of same maps, and you can't know which one to use. I understand. I will add to the docs that the user should move all their maps up one level out of the season directories. I am assuming when you built this lib, you didn't have your maps setup like the client has them when you install it.

Reply:
> lemme know if issue fixed or not

`~` and dll error fixed! Thanks! :D